oracle-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,13 @@
+driverName=oracle
+databaseName=xe
+selectQuery=select * from ${schema}.${table}
+sourceRef=source
+targetRef=target
+sourceTable=ST2
```

Comment:
Please use the full name such as "SourceTable" instead of "ST2". Same for TT2 as well.

Reply:
Done